src/VarDumper.php

```diff
@@ -115,13 +120,18 @@ public function asString(int $depth = 10, bool $highlight = false): string
      * by using the PHP functions {@see serialize()} and {@see unserialize()}.
      *
      * @param bool $format Whatever to format code.
+     * @param array $useVarInClosures Array of variabels used in `use` statement (['$params', '$config'])
+     * @param bool $serializeObjects If it is true all objects will be serialized except objects with closure(s). If it
+     *                               is false only objects of internal classes will be serialized.
      *
+     * @return string A PHP code representation of the variable.
      * @throws ReflectionException
      *
-     * @return string A PHP code representation of the variable.
      */
-    public function export(bool $format = true): string
+    public function export(bool $format = true, array $useVarInClosures = [], bool $serializeObjects = true): string
```

Comment:
I don't think there's a reason to serialize objects in case you can avoid it. Let's remove the option.

Reply:
What `useVarInClosures` is for? Why should I specify it? What if I don't know what I'm exporting?

Reply:
> I don't think there's a reason to serialize objects in case you can avoid it. Let's remove the option.

We need a benchmark.

Reply:
> What `useVarInClosures` is for? Why should I specify it? What if I don't know what I'm exporting?

For forwarding external vars via `use` statment of anonumous function.

Reply:
`serializeObjects` does not work well. `testExportObjectWithClosure` gives:

```php
stdClass#562
(
    [a] => fn () => 1
)
```

and that's not the code that could be executed.

Reply:
> `serializeObjects=false` does not work well. `testExportObjectWithClosure` gives:
> 
> ```
> stdClass#562
> (
>     [a] => fn () => 1
> )
> ```
> 
> and that's not the code that could be executed.

Ok, I'll fix it.

Reply:
Fixed

Reply:
Now let's do benchmark part. If it's in favor of not using serialization, we'll remove the option.

Reply:
> Now let's do benchmark part. If it's in favor of not using serialization, we'll remove the option.

I'll make it today